cmd/end2endtest/main.go

```diff
@@ -35,10 +36,11 @@ type VochainTest interface {
 }
 
 type e2eElection struct {
-	api    *apiclient.HTTPclient
-	config *config
-
-	election      *vapi.Election
+	api      *apiclient.HTTPclient
+	config   *config
+	election *vapi.Election
+	voters   sync.Map
```

Comment:
see the `go vet` warnings; you need to pass around `e2eElection` as a pointer now, otherwise you're copying the `sync.Map` with a mutex, which is probably wrong.

Reply:
Yeah, this is included in the TODO